cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -335,7 +338,7 @@ def start_worker(kube_api, kube_control, auth_control, cni):
         hookenv.log('Waiting for cluster cidr.')
         return
 
-    creds = kube_control.get_auth_credentials()
+    creds = db.get('credentials')
```

Comment:
Again, why cache? I'm worried about the potential for cache drift here.

Reply:
Removed the caching. We shouldn't be storing locally the tokens.